cmd/avalanche-network-runner/control/control.go

```diff
@@ -636,10 +636,10 @@ func saveSnapshotFunc(cmd *cobra.Command, args []string) error {
 
 func newLoadSnapshotCommand() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "load-snapshot snapshot-name",
+		Use:   "load-snapshot snapshot-name [root-data-dir]",
 		Short: "Requests server to load network snapshot.",
 		RunE:  loadSnapshotFunc,
-		Args:  cobra.ExactArgs(1),
+		Args:  cobra.RangeArgs(1, 2),
```

Comment:
root data dir should be passed optional, with a flag, same as start, and other parameter for load snapshot in current main

Reply:
done